cmd/derper/derper.go

```diff
@@ -141,6 +144,15 @@ func writeNewConfig() config {
 	return cfg
 }
 
+func checkMeshKey(k string) (key string, err error) {
+	key = strings.TrimSpace(string(k))
+	if matched, _ := regexp.MatchString(`(?i)^[0-9a-f]{64,}$`, key); !matched {
```

Comment:
Although it's not a performance issue, I think for readability it would be nice to pull this out as an init-time variable:

```go
var validMeshKey = regexp.MustCompile(`(?i)^[0-9a-f]{64,}$`)

// ...
if !validMeshKey.MatchString(key) {
```

Reply:
Can do was just a move from the other location.